plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -1575,6 +1612,39 @@ public void testTypeWidening()
                 .matches("VALUES integer '127', integer '32767', integer '2147483647'");
     }
 
+    @Test
+    public void testTypeWideningNestedSkipping()
```

Comment:
I'm trying to understand, what exactly is being skipped here?
`testTypeWideningNestedSkipping` the name of the method is not helping me.

Reply:
Does it sound better as`testTypeWideningSkippingUnsupportedNestedTypes`? This test asserts that columns of nested types where unsupported widening happens on one of the nested fields are skipped.